### PR TITLE
[[ Bug ]] Fix documentation examples that don't compile

### PR DESCRIPTION
**Pull request by montegoulding**

### CODE REVIEW

**Comment by livecodeali** [comment]
Only noticed because you corrected the type but in the `revBrowserAddJavaScriptHandler` command,

```
 Example:
 local tBrowserID
 put revOpenBrowserCef(the windowId of this stack, "http://www.livecode.com") into tBrowserID
 revBrowserAddJavaScriptHander, tBrowserID, "myLiveCodeHandler"
```

The last line should be 
`revBrowserAddJavaScriptHandler tBrowserID, "myLiveCodeHandler"`

File: docs/dictionary/command/revBrowserAddJavaScriptHandler.lcdoc [hunk 1]
```diff
@@ -24,32 +24,25 @@ instanceID: the ID of the browser instance returned by the revOpenBrowser functi
...
-   answer "Browser ID" && pID && "called myLiveCodeHandler with arguments" && pArg1 && "and" && pArg2
-end myLiveCodeHandler
+    on myLiveCodeHandler pID, pArg1, pArg2
+        answer "Browser ID" && pID && "called myLiveCodeHandler with arguments" && pArg1 && "and" && pArg2
+    end myLiveCodeHandler
 
 Registered within the JavaScript context with the following command:
 
-revBrowserAddJavaScriptHander, tBrowserID, "myLiveCodeHandler"
+    revBrowserAddJavaScriptHander, tBrowserID, "myLiveCodeHandler"
```
Comment (livecodeali) [inline docs/dictionary/command/revBrowserAddJavaScriptHandler.lcdoc]: should be `revBrowserAddJavaScriptHandler tBrowserID, "myLiveCodeHandler"` here too!

**Comment by livecodeali** [comment]
The same typo is present in `revBrowserRemoveJavaScriptHandler`

**Comment by montegoulding** [comment]
Good catch @livecodeali

File: docs/dictionary/command/revBrowserAddJavaScriptHandler.lcdoc [hunk 2]
```diff
@@ -17,39 +17,32 @@ Security: network
 Example:
 local tBrowserID
 put revOpenBrowserCef(the windowId of this stack, "http://www.livecode.com") into tBrowserID
-revBrowserAddJavaScriptHander, tBrowserID, "myLiveCodeHandler"
+revBrowserAddJavaScriptHandler, tBrowserID, "myLiveCodeHandler"
```
Comment (livecodeali) [inline docs/dictionary/command/revBrowserAddJavaScriptHandler.lcdoc]: There's still a comma after the handler name :-)

File: docs/dictionary/command/revBrowserAddJavaScriptHandler.lcdoc [hunk 3]
```diff
@@ -17,39 +17,32 @@ Security: network
...
-   answer "Browser ID" && pID && "called myLiveCodeHandler with arguments" && pArg1 && "and" && pArg2
-end myLiveCodeHandler
+    on myLiveCodeHandler pID, pArg1, pArg2
+        answer "Browser ID" && pID && "called myLiveCodeHandler with arguments" && pArg1 && "and" && pArg2
+    end myLiveCodeHandler
 
 Registered within the JavaScript context with the following command:
 
-revBrowserAddJavaScriptHander, tBrowserID, "myLiveCodeHandler"
+    revBrowserAddJavaScriptHandler, tBrowserID, "myLiveCodeHandler"
```
Comment (livecodeali) [inline docs/dictionary/command/revBrowserAddJavaScriptHandler.lcdoc]: and here

File: docs/dictionary/command/revBrowserRemoveJavaScriptHandler.lcdoc [hunk 4]
```diff
@@ -15,14 +15,14 @@ Platforms: desktop
 Security: network
 
 Example:
-revBrowserRemoveJavaScriptHander, tBrowserID, "myLiveCodeHandler"
+revBrowserRemoveJavaScriptHandler, tBrowserID, "myLiveCodeHandler"
```
Comment (livecodeali) [inline docs/dictionary/command/revBrowserRemoveJavaScriptHandler.lcdoc]: and here

File: docs/dictionary/command/revBrowserAddJavaScriptHandler.lcdoc (hunk 2 above)
Comment (montegoulding) [inline docs/dictionary/command/revBrowserAddJavaScriptHandler.lcdoc]: Aha! I guess I should modify my test to look for code in the description...

Comment (livecodeali) [inline docs/dictionary/command/revBrowserAddJavaScriptHandler.lcdoc]: I guess it's not actually a syntax error though is it? Just an empty first argument.

Comment (montegoulding) [inline docs/dictionary/command/revBrowserAddJavaScriptHandler.lcdoc]: hmpf... still I think it's worthwhile to look for code blocks

**Comment by livecodefraser** [comment]
I like the test, except the bit about having 299 entries that fail!

**Comment by montegoulding** [comment]
Indeed! Hence the WIP label ;-)

**Comment by asayd** [comment]
@montegoulding Hi Monte, finally getting to this after the thread on use-livecode last week. How is the validator implemented? A script only stack? I'm working through it trying to grok your code and would like to try it on some dictionary entries.

**Comment by montegoulding** [comment]
@asayd Howdy! The test is implemented in the LCS test harness. There's a couple of ways we could go to get you setup using it. How keen would you be on getting setup to use git on your local system rather than on GitHub?

**Comment by asayd** [comment]
@montegoulding I'm still a relative noob with git. I've only used github so far, which I find clumsy but usable, but I'm willing to learn how to use it locally. Are you talking about CLI or a local GUI? If GUI, what to you recommend for mac?

**Comment by montegoulding** [comment]
@asayd I think if you are doing more than the occasional docs tweak you will find things considerably quicker working locally. SourceTree is a decent git gui available for Mac and Windows if you are interested steer clear of a CLI. @trevordevore wrote a [blog](http://www.bluemangolearning.com/livecode/2014/07/using-sourcetree-to-monitor-progress-on-livecode/) about getting started in source tree. The screenshots are slightly out dated but you should get the gist. There are a couple of things you will want to do slightly different to the blog post:
- initially clone your fork rather than livecode one which should be `https://github.com/asayd/livecode.git` and be called `origin`
- where he adds the remotes for the team members don't worry about that unless you want to but add a remote called `upstream` which should point to  `https://github.com/livecode/livecode.git`

Now you can double-click on the `upstream/develop` branch and that should create a local branch need `develop` which tracks the main repo rather than your fork. At some point I'd like to write this up for the contributing page as an alternative to the web UI and command line.

**Comment by livecodeali** [comment]
@livecode-vulcan review ok fbcf09f

**Comment by livecode-vulcan** [comment]
💙 review by @livecodeali ok fbcf09f27d4720a27741596ef03a3a9843f8a6d0

**Comment by livecode-vulcan** [comment]
😎 test success fbcf09f27d4720a27741596ef03a3a9843f8a6d0
- try-community-linux-x86: [success](https://vulcan.livecode.com/builders/try-community-linux-x86/builds/2651)
- try-community-mac: [success](https://vulcan.livecode.com/builders/try-community-mac/builds/2661)
- try-community-linux-x86_64: [success](https://vulcan.livecode.com/builders/try-community-linux-x86_64/builds/2646)
- try-community-emscripten: [success](https://vulcan.livecode.com/builders/try-community-emscripten/builds/2662)
- try-community-ios: [success](https://vulcan.livecode.com/builders/try-community-ios/builds/2668)
- try-community-win-x86: [success](https://vulcan.livecode.com/builders/try-community-win-x86/builds/2674)
- try-community-android-armv6: [success](https://vulcan.livecode.com/builders/try-community-android-armv6/builds/2686)

**Pull request merged**